### DISCUSSION TITLE
Using ESP32 as BT "sender"

### DISCUSSION THREAD

**Discussion by laverdanny** [pschatzmann/ESP32-A2DP#405] [Q&A] [answered]
hi
Ive found in this repository a way to create a full audio receiver using internal DAC

Is there a way to get analog audio, encode it in stereo PCM and sent it over BT using (almost) only ESP32 resources ?

Regards

**Comment by pschatzmann** [comment, marked as answer]
Sure as documented in the Readme of this project!
You can also find some examples and even more using the [AudioTools library](https://github.com/pschatzmann/arduino-audio-tools).
